packages/shared/src/components/modals/PostUpvotersModal.tsx

```diff
@@ -0,0 +1,66 @@
+import request from 'graphql-request';
+import React, { ReactElement } from 'react';
+import { useInfiniteQuery } from 'react-query';
+import { PostUpvotesData, POST_UPVOTES_BY_ID_QUERY } from '../../graphql/posts';
+import useFeedInfiniteScroll from '../../hooks/feed/useFeedInfiniteScroll';
+import { apiUrl } from '../../lib/config';
+import { ModalProps } from './StyledModal';
+import { UpvotedPopupModal } from './UpvotedPopupModal';
+
+interface PostUpvotersModalProps extends ModalProps {
+  postId: string;
+}
+
+export function PostUpvotersModal({
```

Comment:
Please apply here the same comments from the other file.
I actually have an idea for a component that gets only the parameters for the GraphQL request and then handles the pagination for you. Instead of creating two different components. I know I suggested the two components but now that I see how much they resemble, we need to think about it.

Reply:
This will be solved when I create the reusable modal that incorporates `useInfiniteScroll`, I have two thoughts in mind on how to do this, I will try to weigh which one would be more beneficial, then I will explain why.